docs/src/pages/_app.page.tsx

```diff
@@ -86,11 +87,17 @@ function MyApp({ Component, pageProps }) {
 
       <div className={isHomepage ? `docs-home` : ''}>
         <ThemeProvider theme={baseTheme} colorMode={colorMode}>
-          <GlobalNav
-            rightLinks={RIGHT_NAV_LINKS as NavMenuItem[]}
-            leftLinks={LEFT_NAV_LINKS as NavMenuItem[]}
-            currentSite="UI Library"
-          />
+          {
+            /* Remove after Global Nav launch START */
+            isGlobalNavEnabled /* Remove after Global Nav launch END */ ? (
```

Comment:
```suggestion
            isGlobalNavEnabled ? (  /* Remove after Global Nav launch END */
```

nit: the reason why we need to make it accurate is because potentially we can create a script to remove everything between the comments. Once we have more feature flags or a/b tests, we can do that 😁